package.json

```diff
@@ -33,6 +33,7 @@
         "@octokit/rest": "latest",
         "@types/chai": "latest",
         "@types/convert-source-map": "latest",
+        "@types/fs-extra": "^9.0.13",
```

Comment:
Noticed this while cleaning up our dependencies, as I was hoping we could just use `fs/promises` instead (unfortunately not).